.readthedocs.yaml

```diff
@@ -0,0 +1,40 @@
+version: 2
+
+formats:
+  - htmlzip
+
+sphinx:
+  configuration: docs/conf.py
+
+build:
+  os: ubuntu-24.04
+  tools:
+    python: "3.13"
+  apt_packages:
+    - graphviz
+  jobs:
+    post_checkout:
+      # Skip docs build if the commit message contains "skip ci"
+      - (git --no-pager log --pretty="tformat:%s -- %b" -1 | grep -viq "skip ci") || exit 183
+      # Skip docs build if there are no changes related to docs
+      - |
+        if [ "$READTHEDOCS_VERSION_TYPE" = "external" ] && git diff --quiet origin/main -- bindings/ docs/ include/ mlir/include/ python/ .github/contributing* .github/support* .readthedocs.yaml;
+        then
+          exit 183;
+        fi
+      # Unshallow the git clone and fetch tags to get proper version information
+      - git fetch --unshallow --tags
+    pre_build:
+      # Set up uv
+      - asdf plugin add uv
+      - asdf install uv latest
+      - asdf global uv latest
+      - curl -LsSf https://github.com/munich-quantum-software/setup-mlir/releases/latest/download/setup-mlir.sh | bash -s -- -v f8cb7987c64dcffb72414a40560055cb717dbf74 -p $HOME/mlir
+      - uv pip install "mqt.core~=3.3.3" "pennylane-catalyst~=0.13.0"
```

Comment:
```suggestion
```
This step is redundant and not needed as  both packages are installed as part of the package build.